weave-js/src/components/PagePanelComponents/Home/HomeCenterBrowser.tsx

```diff
@@ -436,10 +440,14 @@ export const ActionCell = <RT extends CenterBrowserDataType>(props: {
                 <CenterTableActionCellAction
                   key={'' + j + '_' + k}
                   onClick={e => {
+                    if (a.disabled != null && a.disabled(props.row)) {
+                      return;
```

Comment:
Probably doesn't matter in practice, but I'm not sure why the stopPropagation is below - presumably we want to stop event propagation for disabled too? That should happen if you change this to `return false`